### DISCUSSION TITLE
Break door script

### DISCUSSION THREAD

**Discussion by SOF-Ice** [Vdauphin/HeartsAndMinds#1565] [Question and answer] [answered]
Hello Vdauphin,

Where can I find the script that defines the type of object needed to open a closed door? As the witecutter and claymore 

iwould like to add a new way to break a door lock as using a shotgun

**Comment by Vdauphin** [comment, marked as answer]
Hello,

https://github.com/Vdauphin/HeartsAndMinds/blob/34f5a564cdb0be456b30bb86d21c34f90d5f641c/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf#L240

https://github.com/Vdauphin/HeartsAndMinds/blob/34f5a564cdb0be456b30bb86d21c34f90d5f641c/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/eh/server.sqf#L57-L61

Cheers
